add use cases and include traffic nature (#34)
SpencerDawkins/net-collab-rqmts: draft-kwbdgrr-tsvwg-net-collab-rqmts.md

Examples: live broadcast, on-demand video streaming

Use cases:

>   1. In loss-prone networks or during reactive policy events, retransmissions cause immense delay. All packets being treated the same can have challenges in efficiently handling/forwarding data. Today, there is no way to identify packets which are less important and/or loss-tolerant to prioritize packets in challenging networks and/or during reactive events.

Reviewer: ```suggestion
  1. In loss-prone networks or during reactive policy events, retransmissions cause long delays. All packets being treated the same can have challenges in efficiently handling/forwarding data. Today, there is no way to identify packets which are less important and/or loss-tolerant to prioritize packets in challenging networks and/or during reactive events.
```

Determination of such preferences is outside of the scope of this document.

Requirements:  

  REQ-CLIENT-DECIDES: The receiving client determines importance of packets it receives, as the client may have changing needs over time.

Use cases:

  1. Dynamic changes to priority based on user activity is not possible today. For example, audio packets having the same priority when a user mutes the audio locally, or change in priority during times of emergency where video streaming applications share the same priority as SOS signals.
  
  2. A user prioritizing video over audio while playing an interactive game.

>   3. User's foreground application should receive priority over background tasks. For eg., a user is typing in a document while playing a media in the background within the same session.

Reviewer: ```suggestion
  3. User's foreground application should receive priority over background tasks. For example, a user is typing in a document while playing a media in the background within the same session.
```

It is out of the scope of this document to discuss setups (e.g., 3GPP PDU Sessions) where network attachments with Guaranteed Bit Rate (GBR) for specific flows is provided.

Requirement:
  REQ-SCOPED-METADATA: Means to characterize the scope of a shared metadata for the sake of better interoperability should be supported.

## Application Interference {#app-interference}

Applications that have access to a resource-quota information may adopt an aggressive behavior (compared to those that don't have access) if they assumed that a resource-quota like metadata is for the application, not for the host that runs the applications.

This is challenging for home networks where multiple hosts may be running behind the same CPE, with each of them running a video application. The same challenge may apply when tethering is enabled.

Requirement:

>   REQ-SIGNAL-EXPOSURE-FAIRNESS: Means to expose the signal independent of the application should be considered.

Reviewer: ```suggestion
  REQ-SIGNAL-EXPOSURE-FAIRNESS: Means to expose the signal independent of the application should be considered. An example of such exposure is OS APIs.
```

REQ-FRAME-END: Indicate packet containing end of media frame.

> ### Identification of traffic types {#TrafficType}

Reviewer: ```suggestion
### Identification of Traffic Type without Disclosure the Application {#TrafficType}
```
